pkg/config/polling_manager_test.go

```diff
@@ -192,6 +211,7 @@ func TestPollingGetOptimizelyConfig(t *testing.T) {
 	configManager.SyncConfig(mockDatafile2)
 	optimizelyConfig = configManager.GetOptimizelyConfig()
 	assert.Equal(t, "43", optimizelyConfig.Revision)
+	//mockRequester.AssertExpectations(t)
```

Comment:
@msohailhussain , not sure why I could not check for requester assert expectation. Can you take a look ?

Reply:
I don't know how it was working before. When SyncConfig is called with payload in the argument, it means no `Get` call should be asserted. I have removed explicit  `SyncConfig` call and now dependent on `Start` scheduled call. Very first call is called immediately, so no major wait in this unit test.